lib/gradient.ex

```diff
@@ -133,7 +133,6 @@ defmodule Gradient do
   defp maybe_specify_forms(forms, opts) do
     unless opts[:no_specify] do
       forms
-      |> put_source_path(opts)
```

Comment:
As mentioned in #138 this line is not necessary because before `maybe_specify_forms/2` is called, we already call `put_source_path`.